src/lib/emoticons.ts

```diff
@@ -0,0 +1,254 @@
+// this list was generated by removing the word/numbers in the following list:
+// https://github.com/cjhutto/vaderSentiment/blob/master/vaderSentiment/vader_lexicon.txt
+// and copying them into emoticons.txt and then extracting the initial characters up to the
+// first space using: awk '{print $1}' emoticons.txt
+
+const rawEmoticons = `
+$:
+%)
+%-)
+&-:
+&:
+(
```

Comment:
This one seems like it might cause problems.

Reply:
It is escaped on line 252.

Reply:
Ah - I see, you mean just the bare paren.  You are right, I'll remove that.

Reply:
@tealefristoe I removed the bare open paren in follow on commit: https://github.com/concord-consortium/storyq-codap-plugin/pull/19/commits/05d55ee9a63311b6ea8d59d5b98c0fbdf9fbb7b7